### PR TITLE
Crew Website

### PR DESCRIPTION
**Pull request by offlinemark**
Figured I should contribute so I threw this together this afternoon. Content definitely needs a lot of work as does layout, so if someone has something better or people don't like this one, it's totally fine if we don't use it. Just putting it out there.

### CODE REVIEW

**Comment by pranav** [comment]
Waiting on design review from @ali

File: index.html [hunk 1]
```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+	<link href='http://fonts.googleapis.com/css?family=Iceland' rel='stylesheet' type='text/css'>
```
Comment (pranav) [inline index.html:5]: I say change the font. I don't really like it.

**Comment by wylieconlon** [comment]
Screenshot attached:

Screen Shot 2013-02-09 at 4 55 30 PM

This is pretty good, but needs more line-spacing on the paragraphs. I would also suggest making the entire width smaller to reduce the length of those lines, because it's hard to read.

**Comment by ali** [comment]
Yo Mark! We should do a design review meeting with the rest of the web team! :+1:

**Comment by offlinemark** [comment]
Yeah sure, that would be fine with me. I also feel like getting the content down is pretty important and should be discussed.

**Comment by ali** [comment]
Closing this (will re-open later)

**Pull request closed without merging**